README.md

```diff
@@ -37,75 +39,75 @@ Desktop version day2
   
   Git,Github and Visual Studio Code
 
- 💻 Getting Started 
+ ## 💻 Getting Started 
  
 To get a local copy up and running, follow these steps.
 
- Prerequisites
+##  Prerequisites
 
 In order to run this project you need:
 VSCode 
 version control system
-Setup
+
+## Setup
+
 Clone this repository to your desired folder:
 
   cd my-folder
   git clone git@github.com:theresetuyi/porfolio.git
 
 
-Install
+### Install
 
 Install this project with:
 
 
   cd my-project
   npm install
 
-
-Usage
+## Usage
 
 To run the project, execute the following command:
 If you have live server extension
 
-👥 Authors 
+## 👥 Authors 
 
 - GitHub: https://github.com/theresetuyi
 - Twitter:https://twitter.com/THERESETUYISAB2
-- LinkedIn:https://www.linkedin.com/in/therese-theddy-tuyisabe
+- LinkedIn:https://www.linkedin.com/in/therese-theddy-tuyisabe-249820203/
 
-👥 Authors 
+## 👥 Authors 
 
-- GitHub:https://github.com/Gardimy
-- Twitter:https://twitter.com/
-- LinkedIn:https://www.linkedin.com/in/gardimy-charles-40b795212/
+- GitHub: https://github.com/Hamdullah-Andar
+- Twitter: https://twitter.com/HamdullahAndar
+- LinkedIn: https://www.linkedin.com/in/hamdullah-andar-98b35894/
 
- 🔭 Future Features 
+ ## 🔭 Future Features 
  
 -I will like to add new technology
 -I will lake to add js code 
--I would still let it be public
 
-🤝 Contributing
+## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page]
 https://github.com/theresetuyi/porfolio/issues
 
 
-⭐️ Show your support 
+## ⭐️ Show your support 
 
  Write a message to encourage readers to support your project
 
 If you like this project type ❤❤
 
- 🙏 Acknowledgments 
+ ## 🙏 Acknowledgments 
 
  Give credit to everyone who inspired your codebase.
 
 I would like to thank my partner 
 
-📝 License 
+## 📝 License 
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 add the LICENSE file to your repo and make sure that the MIT link works as expected for a professional README file.